.github/workflows/update-wpt.yml

```diff
@@ -0,0 +1,76 @@
+name: WPT update
+
+on:
+  schedule:
+    # Run once a week at 12:00 AM UTC on Sunday.
+    - cron: 0 0 * * *
+  workflow_dispatch:
+    inputs:
+      subsystems:
+        description: Subsystem to run the update for
+        required: false
+        default: '["url"]'
+
+permissions:
+  contents: read
+
+jobs:
+  wpt-subsystem-update:
+    if: github.repository == 'nodejs/node' || github.event_name == 'workflow_dispatch'
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        subsystem: ${{ fromJSON(github.event.inputs.subsystems || '["url"]') }}
+
+    steps:
+      - uses: actions/checkout@v4.1.7
+        with:
+          persist-credentials: false
+
+      - name: Install @node-core/utils
+        run: |
+          npm install -g @node-core/utils
+
+      - name: Calculate new version for WPT
+        run: |
+          # Hypothetically, you can get the latest version from a specific file or git log
+          new_version=$(git log -1 --pretty=format:"%h")
+          echo "new_version=$new_version" >> $GITHUB_ENV
```

Comment:
You can only obtain the new version after `git node wpt` is done. And you can obtain the new version with `jq` extraction using the `test/fixtures/wpt/versions.json` file as input